source/deployment/ldap-group-constrained-team-channel.rst

```diff
@@ -17,42 +17,58 @@ Similarily on private channels that are managed by synchronized groups, users ou
 2. Ensure **Team Settings > General > Allow any user with an account on this server to join this team** is set to ``No``. 
 3. Convert the team to have its membership managed by synchronized groups by running the `group team enable CLI command <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-group-team-enable>`_.
 
 To manage membership of a private channel with synchronized groups: 
 
-1. Ensure there is at least one group already associated to the channel. You can view and add default channels to a group via **System Console > User Management > Groups > Group Configuration**. Please see more information on adding default teams and channels `here <https://docs.mattermost.com/deployment/ldap-group-sync.html#add-default-teams-or-channels-for-the-group>`_. Additionally, you can use the CLI tool to view if there is already a group associated to the channel by running the `group channel list CLI command <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-group-team-list>`_.
+1. Navigate to **System Console > User Management > Channels**. Select the channel you want to manage with group synchronization.  
+2. Under **Channel Management**, enable **Sync Group Members**. Please ensure the channel is set to ``private``. 
+3. Add one or more groups to the channel. If there are groups already associated to default users into the channel, they will already be present.  
+4. Review the notice in the footer of the screen for any users that are not part of groups who will be removed from the channel on the next synchronization.
+5. Click save. Members will be updated on the next scheduled AD/LDAP synchronization. 
+
+Alternatively you can use the CLI tool to set a private channel to be managed by groups: 
 
+1. Ensure there is at least one group already associated to the channel. You can view and add default channels to a group via **System Console > User Management > Groups > Group Configuration**. Please see more information on adding default teams and channels `here <https://docs.mattermost.com/deployment/ldap-group-sync.html#add-default-teams-or-channels-for-the-group>`_. Additionally, you can use the CLI tool to view if there is already a group associated to the channel by running the `group channel list CLI command <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-group-team-list>`_.
```

Comment:
Wondering what happens if there isn't one group?

Reply:
For the CLI, I believe you will just be in a weird scenario where the policy is set, but there is no one in the channel/team because you didn't add any groups to it.  In the UI, we prevent you from changing the management to synced to groups without having a group.  @mkraft can you please confirm for the CLI?

Reply:
The CLI prevents one from enabling group-constraints without at least one associated group (same as the system console web UI).

Reply:
Thanks for clarifying! Glad it is the same in both :)